docs/shader-modules/project.md

```diff
@@ -95,15 +95,27 @@ Projects world space coordinates to clipspace, which can be assigned to `gl_Posi
 `vec4 project_to_clipspace(vec4 position)`
 
 
-### project_pixels_to_clipspace
+### project_pixel_to_clipspace
 
 Converts the given number of pixels to a clipspace offset that can be added to a clipspace position (typically added to values returned by `project_to_clipspace`).
 
-`vec4 project_pixels_to_clipspace(vec2 pixels)`
+`vec4 project_pixel_to_clipspace(vec2 pixels)`
 
 * `pixels` (`vec2`) - adjustment in logical pixels. Returns values in clip space offsets that can be added directly to `gl_Position`
 
 
+### project_pixel_to_worldspace
```

Comment:
We are projecting FROM worldspace, not TO worldspace. We really need a clear name for the internal shader working space, and I would dearly prefer if that name did not include the word "world" :)